README.md

```diff
@@ -1,12 +1,26 @@
 ## ALiAS: Amity Linux Assistance Sapience
-- The following repo contains the source code for the official website of Amity Linux Assistance Sapience. 
+
+ [![HitCount](http://hits.dwyl.io/asetalias/asetalias.github.io.svg)](http://hits.dwyl.io/asetalias/asetalias.github.io) [![GitHub license](https://img.shields.io/github/license/asetalias/blog.svg?style=flat-square)](https://github.com/asetalias/asetalias.github.io/blob/master/LICENSE) 
+![GitHub issues](https://img.shields.io/github/issues-raw/asetalias/asetalias.github.io.svg) ![GitHub repo size](https://img.shields.io/github/repo-size/asetalias/asetalias.github.io.svg)
+![Twitter Follow](https://img.shields.io/twitter/follow/asetalias.svg?style=social)
+
+
+- The following repository contains the source code for the official website of Amity Linux Assistance Sapience. The largest open-source community of Amity University in Noida and now also in Lucknow. 
 - The website is hosted on gh-pages itself. 
-- This Website uses Staticjinja to build top-level HTML files. Follow Instructions to modify the HTML Files.
+- This Website uses Staticjinja to build top-level HTML files. Follow Instructions to modify the HTML files.
+
+<img src="https://user-images.githubusercontent.com/22801822/67155846-3d12c500-f334-11e9-9125-dbf32ea573a9.png" height=480>
 
-## Contribute 
-- Feel free to open an issue or recommend any changes which you want to see in the website. 
+## Contribute to the Website 
+- Feel free to open an issue or recommend any changes which you want to see on the website. 
 - Adhere to the guidelines mentioned below if you want to contribute. 
-- To Contribute, check out official [wiki](https://github.com/asetalias/asetalias.github.io/wiki)
+- Before contributing, be sure to check out the official [wiki](https://github.com/asetalias/asetalias.github.io/wiki)
+
+## Contirbute to ALiAS Writes's blog!
+
+ALiAS has its own blog now, [ALiAS Writes](https://asetalias.in/blog). If you like your contributions to be more content, than technical then we we would love to see your blogs be published on ALiAS Writes. [Read more about it here on How to Contibute](https://github.com/asetalias/blog/blob/master/README.md). 
```

Comment:
```suggestion
ALiAS has its own blog now, [ALiAS Writes](https://asetalias.in/blog). If you like your contributions to be more content-centric than technical, we would love to see your blogs on ALiAS Writes. Read more about How to Contribute [here](https://github.com/asetalias/blog/blob/master/README.md). 
```

Reply:
I feel something like content-centric, content-oriented or content-based is better than just content